aion/web/elm/src/Update.elm

```diff
@@ -468,10 +471,14 @@ update msg model =
                 )
 
         ReceiveDisplayQuestion _ ->
-            { model | roomState = QuestionDisplayed } ! []
+            { model
+                | roomState = QuestionDisplayed
+                , progressBar = model.progressBar |> withProgress 0 |> withRunning Uninitialized |> withStart 0
```

Comment:
Is there a way to format pipes in new line each?

Reply:
Absolutely, I could do it, I wanted to refrain from building a visual tree and update each part of the model in one line. I can make it multiline though